.gitignore

```diff
@@ -42,3 +42,5 @@ local.properties
 # Sandbox stuff
 _sandbox
 
+signing.properties
+droidconit.jks
```

Comment:
@rock3r let me fix that for you

![MAGIC](https://cloud.githubusercontent.com/assets/2116164/6994307/3557e30e-db15-11e4-99e5-0c234d3b56d3.gif)